index.php

```diff
@@ -29,6 +29,8 @@
         'avatarurls'          => false,
         'hideforms'           => false,
         'expand'              => false,
+        'allowlinks'          => true,
```

Comment:
I agree with these defaults. This is likely the expected behaviour; and if not desired it can easily be disabled :+1: